src/database/operations/load/loadBundle.ts

```diff
@@ -1,24 +1,18 @@
+import { db } from '../../db';
 import { loadTableMetrics } from './loadTableMetrics';
-import { loadObjects } from './loadObjects';
 import type { TinyPgParams } from 'tinypg';
-import type { ValidateFunction } from 'ajv';
 import type { Bundle } from '../../../types';
 
 export const loadBundle = async <T extends object>(
 	tinyPgQueryName: string,
 	tinyPgQueryParameters: TinyPgParams,
 	tableName: string,
-	entryValidator: ValidateFunction<T>,
```

Comment:
If "load" here means "read from the database and write into node memory", no problem. If "load" means "read from node memory and write to the database" we'd want the validation, right? I am pretty sure this is reading from the DB, though.

Reply:
yeah -- it's reading *from* the DB!